src/lib/connectors/repos/billing-batches.js

```diff
@@ -30,6 +29,15 @@ const findPage = async (page, pageSize) => {
   return paginatedEnvelope(result);
 };
 
+const findByStatus = async status => {
+  const batches = await BillingBatch
+    .forge()
+    .where({ status })
+    .fetchAll({ withRelated: ['region'] });
+
+  return batches.toJSON();
+};
+
```

Comment:
Could we use Bookshelf to find the single pending batch in the region directly, e.g:
```
const findOneByRegionAndStatus = async (regionId, status) => {
  const batch = await BillingBatch
    .forge()
    .where({ region_id : regionId, status })
    .fetch({ withRelated: ['region'], require: false });

  return batch ? batch.toJSON() : null;
};
```